src/main/kotlin/no/digdir/fdk/searchservice/kafka/KafkaRemovedEventCircuitBreaker.kt

```diff
@@ -49,39 +49,50 @@ open class KafkaRemovedEventCircuitBreaker(
 
         val event = record.value()
         try {
-            val timeElapsed = measureTimedValue {
+            val (deleted, timeElapsed) = measureTimedValue {
+                var del = false
                 event.let {
```

Comment:
Og let-funksjonen her gjør ingenting annet enn å endre navnet til variabelen fra `event` til `it`, så hadde vært bedre å bare fjerne den og endre bruk av `it` til `event`